deployer/aws-lambda/content-origin-request/index.js

```diff
@@ -136,6 +138,11 @@ exports.handler = async (event) => {
     return redirect(`/${locale}${path || "/"}` + qs);
   }
 
+  // Redirect moved pages (see `_redirects.txt` in content/translated-content).
+  if (typeof REDIRECTS[requestURILowerCase] == "string") {
+    return redirect(REDIRECTS[requestURILowerCase]);
+  }
```

Comment:
Hm, this only redirects HTML requests, but the JSON requests (`index.json`) should probably be redirected as well. 🤔

Reply:
✅ in f91e2beb1c38fed43a7c255e02484399fe5f2bd2.